index.bs

```diff
@@ -78,8 +78,8 @@ Ambient Light Sensor provides information about lighting conditions near
 the device environment. Potential privacy risks include:
 
   - Profiling. Ambient Light Sensor can leak information about user's use
-    patterns and surrounding. This information can be used to enhance user
-    profiling and behavioral analysis.
+    patterns and surrounding cross-origin. This information can be used to
```

Comment:
@lknik, I'm trying to understand what is the concrete suggestion you are trying to make. Are you suggesting we remote the word "cross-origin", i.e. revert commit https://github.com/w3c/ambient-light/commit/77ebda1947379ef6ec98838e0fa97e9677cf20d2 ?